java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JObject.java

```diff
@@ -280,6 +282,13 @@ public GeneratorResult generateJava() {
 
       objField.createGetter().addAnnotation("com.fasterxml.jackson.annotation.JsonAnyGetter");
       objField.createSetter().addAnnotation("com.fasterxml.jackson.annotation.JsonAnySetter");
+
+      MethodDeclaration additionalSetter = clz.addMethod("setAdditionalProperties", Modifier.Keyword.PUBLIC);
```

Comment:
@andreaTP I mean this would need to change to singular as well.

Reply:
@shawkins that's what I changed to generate the corresponding output everywhere else.